README.md

```diff
@@ -42,8 +42,7 @@ Start [here](docs/setup/install.md).
 - [Okta](docs/setup/config/okta.md) - users, groups, organizations, roles, applications, factors, trusted origins, reply URIs
 - [Github](docs/setup/config/github.md) - repos, branches, users
 - [DigitalOcean](docs/setup/config/digitalocean.md)
-- [Microsoft Azure](docs/setup/config/azure.md) -  CosmosDB, SQL, Storage, Virtual Machine
-
```

Comment:
We should have a line space between `## Supported platforms` and `## Usage` sections